tools/setup

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+
+ROOT=$(dirname $0)/..
+
+# 1. Install nvm
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.34.0/install.sh | sh 
+
+# 1.1 ensure nvm is in your current process
+source ~/.nvm/nvm.sh
+
+# 2. Install node
+nvm install 10
```

Comment:
If there's a version in package.json or .travis.yml you might consider using that?  Or find a way to make this value coherent?

Reply:
Added .nvmrc. 

Otherwise, we should investigate this thread: https://github.com/nvm-sh/nvm/issues/651